.travis.yml

```diff
@@ -35,12 +35,12 @@ env:
     - PATH=${GOPATH}/bin:${GOROOT}/bin:${PATH}
   matrix:
     - TEST_SUITE=EC
-    - TEST_SUITE=EC SDS_BRANCH=4.2.x
-    - TEST_SUITE=EC SDS_BRANCH=4.x
+    - TEST_SUITE=EC SDS_BRANCH=5.x
+    - TEST_SUITE=EC SDS_BRANCH=master
```

Comment:
"master" is the default branch when no branch is specified. By specifying it here, the tests with this branch will run twice.